Sources/LogtoClient/Types/LogtoConfig.swift

```diff
@@ -11,7 +11,26 @@ import Logto
 public struct LogtoConfig {
     let endpoint: String
     let clientId: String
-    let scope: ValueOrArray<String>? = nil
-    let resource: ValueOrArray<String>? = nil
-    let usingPersistStorage: Bool = false
+    let scope: ValueOrArray<String>?
+    let resource: ValueOrArray<String>?
+    let usingPersistStorage: Bool
+
+    var computedScopes: [String] {
+        LogtoUtilities.withReservedScopes(scope)
+    }
+
+    // Have to do this in Swift
+    init(
+        endpoint: String,
+        clientId: String,
+        scope: ValueOrArray<String>? = nil,
+        resource: ValueOrArray<String>? = nil,
+        usingPersistStorage: Bool = false
+    ) {
+        self.endpoint = endpoint
+        self.clientId = clientId
+        self.scope = scope
```

Comment:
just want to confirm there will be two instance `computedScopes` and original scope under the LogtoConfig?

Reply:
i have a better idea

Reply:
updated, PTAL